tests/unit/modules/test_win_disk.py

```diff
@@ -65,12 +55,26 @@ def test_usage(self):
         self.assertDictEqual(
             win_disk.usage(),
             {
-                "A:\\": {
+                "C:\\": {
+                    "available": None,
```

Comment:
Either just check for the existence of these keys or don't check them at all.  The values may be set if the disk actually exists, but this test isn't actually really testing these values.